plugins/services/src/js/components/ServiceConfigDisplay.js

```diff
@@ -87,13 +87,16 @@ class ServiceConfigDisplay extends React.Component {
   }
 
   render() {
+    const {appConfig, errors, handleEditClick} = this.props;
+
     return (
       <ConfigurationMap>
         {this.getRootErrors()}
         <MountService.Mount
-          type={this.getMountType()}
-          errors={this.props.errors}
-          appConfig={this.props.appConfig} />
+          appConfig={appConfig}
+          errors={errors}
+          handleEditClick={handleEditClick}
```

Comment:
I would vote for `onEditClick`, for the property hame. `handle` usually refers to the function that handles it.